components/button/_buttons.scss

```diff
@@ -158,3 +158,67 @@
     display: none;
   }
 }
+
+.btn--mobile {
+  position: relative;
+  overflow: hidden;
+  margin: 3px 0 0;
+  padding: 0;
+  width: 28px;
+  height: 28px;
+  text-indent: -9999px;
+  background: transparent;
+  border: transparent;
+  border-radius: 0;
+  &:focus, &:hover {
+    background-color: transparent;
+    border-color: transparent;
+  }
+  span {
```

Comment:
Think we need to follow the convention of using BEM in the pattern library, entity selectors are discouraged and would be better to use an element selector inside this modifier.

Reply:
Yep good point.